.eslintrc.json

```diff
@@ -39,6 +46,7 @@
         "radix": ["error", "as-needed"],
         "no-shadow": "warn",
         "no-restricted-globals": "warn",
-        "react/no-array-index-key": "warn"
+        "react/no-array-index-key": "warn",
+        "import/extensions": "off"
```

Comment:
I have this rule as off since when attempting to add extensions to the import file paths VSCode yells at me like so

<img width="936" alt="Screen Shot 2023-09-04 at 6 16 55 PM" src="https://github.com/tally-team/tally-frontend/assets/24977851/077645a2-4274-422e-8e16-3b15a3a4ffff">

I can create a follow up ticket to investigate this, but I have this off atm to hopefully also unblock others.

Reply:
Sounds GTM. Can we include the ticket # and add it in a comment with a description here?

Reply:
i added a todo comment about this :D issue here: https://github.com/tally-team/tally-frontend/issues/54